source_mesh.py

```diff
@@ -324,31 +329,35 @@ def create_vertices(vmec, mbc, num_s, num_theta, num_phi):
             vertex.
     """
     # Compute total number of vertices
-    num_verts = (num_phi - 1) * (1 + ((num_s - 1) * (num_theta - 1)))
+    if tor_ext == 360.0:
+        num_verts = (num_phi - 1) * (1 + ((num_s - 1) * (num_theta - 1)))
+    else:
+        num_verts = num_phi * (1 + ((num_s - 1) * (num_theta - 1)))
     # Initialize n x 3 array of vertex coordinates
     verts = np.zeros((num_verts, 3))
     # Initialize array of vertex closed flux surface indices
     verts_s = np.zeros(num_verts)
 
     # Compute regular spacing between toroidal angles in confinement space to
     # be included in mesh
-    delta_phi = 2*np.pi/(num_phi - 1)
+    if tor_ext == 360.0:
```

Comment:
I think it makes sense to deal with both impacts of `tor_ext == 360` in the same place, if possible.